src/Npgsql/NpgsqlBuffer.cs

```diff
@@ -618,5 +618,71 @@ struct BitConverterUnion
         }
 
         #endregion
+
+        #region Postgis
+        //  Postgis returns bytes in ndr representation.
+        internal int PostgisReadInt32()
```

Comment:
Is it more correct to check the byte specifying the byte order, or does PostGIS always do little endian?